BEIMA.Client/src/pages/Users/AddUserPage.js

```diff
@@ -35,29 +42,53 @@ const AddUserPage = () => {
   
   function checkPassword(){
     let passwordErrors = {};
+    let requirements = new RegExp("^(?=.*[a-z])(?=.*[A-Z])(?=.*\\d)(?=.*[-+_!@#$%^&*.,?]).+$");
```

Comment:
Should this be in a constants file?

Reply:
Yeah, that's probably a good place for it

Reply:
Added it to the constants file